Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -513,18 +535,48 @@ private class ParameterSetNames
         [Parameter(Mandatory = false, Position = 4, ParameterSetName = ParameterSetNames.Csv)]
         public string[] Column { get; set; }
 
+        /// <summary>
+        /// <para type="description">
+        ///  If true, uploads the object at ImportFilePath to a new Google Cloud Storage bucket, then imports it
+        ///  into the instance. Otherwise, imports the file at the Google Cloud Storage location
+        ///  specified by ImportFilePath
+        /// </para>
+        /// </summary>
+        [Parameter]
+        public SwitchParameter UploadLocalFile { get; set; }
```

Comment:
This seems unnecessary. Can't we just check if the ImportFilePath is prefixed with "gs://" and if so, we know it is an existing GCS object, otherwise use the local-file path route?